travis_scripts/trigger-dockerhub.sh

```diff
@@ -25,7 +25,7 @@ if [[ -z "$DOCKERHUB_TOKEN" ]] ; then
     exit 1
 fi
 
-if [[ -n "$COVERAGE" ]] ; then
+if [[ -n "$COVERAGE" && $COVERAGE -ne 0 ]] ; then
```

Comment:
So by default it gets set to `0`, and if `$COVERAGE` is already set to `1`, it gets changed to `"coveralls"`.

This means the test should be good, but I'm happy to change that to a string comparison, it shouldn't make any difference.